R/net.mod.infection.R

```diff
@@ -115,9 +121,10 @@ infection.net <- function(dat, at) {
 
   ## Save incidence vector
   if (at == 2) {
-    dat$epi$si.flow <- c(0, nInf)
+    dat <- set_epi(dat, "si.flow", 1, 0)
+    dat <- set_epi(dat, "si.flow", 2, nInf)
   } else {
-    dat$epi$si.flow[at] <- nInf
+    dat <- set_epi(dat, "si.flow", at, nInf)
```

Comment:
No, just this line

Reply:
@smjenness for this, are we wanting epi trackers at step 1 to record NA instead of 0?

Reply:
yes

Reply:
Syntax fixed.